### PR TITLE
merge stable

### PR DESCRIPTION
**Pull request by ibuclaw**

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

**Comment by ibuclaw** [comment]
Build failure is due to CI erroneously cloning druntime @CyberShadow.  This is preventing dlang.org site from being updated + release announcement -> Just merge? Or does DAutoTest need updating to stop doing this?

**Review by CyberShadow — CHANGES_REQUESTED** [review]
Uh oh, the DMD docs fell off

File: posix.mak [hunk 1]
```diff
@@ -431,10 +431,9 @@ ${GENERATED}/modlist-prerelease.ddoc : tools/modlist.d ${STABLE_DMD} $(DRUNTIME_
 		$(addprefix --dump , object std etc core dmd rt core.internal.array core.internal.util) >$@
```
Comment (ibuclaw) [inline posix.mak:431]: This looks like the problem @CyberShadow (`modlist-${LATEST}.ddoc` recipe has a different dmd/src path).

Comment (CyberShadow) [reply posix.mak:431]: Yes, well spotted!

Comment (ibuclaw) [reply posix.mak:431]: Actually, everything in the make files that uses `DMD_LATEST_DIR` needs to be aligned up with the use of `DMD_DIR`.

Comment (CyberShadow) [reply posix.mak:431]: Good idea, updated all the ones I could find.

Comment (ibuclaw) [reply posix.mak:431]: Looks much better now! :tada:

**Review by CyberShadow — APPROVED** [review]
Yep, looks good!

**Pull request merged**